app/routes/helpers.py

```diff
@@ -0,0 +1,104 @@
+from app.models.customer import Customer 
+from app.models.video import Video 
+from flask import request
+
+# helper methods for validation/error-checking 
+def id_is_valid(id, object_type):
+    '''
+    returns two values: an object and None (if no error is present),
+    or an "invalid" string and an error (if error is present) 
+    '''
+    if not id.isnumeric(): 
+        return "invalid", ("", 400) 
+    
+    if object_type == "customer":
+        object = Customer.query.get(id) 
+    elif object_type == "video":
+        object = Video.query.get(id)
+
+    name = object_type.capitalize()
+    if not object:
+        return "invalid", ({"message": f"{name} {id} was not found"}, 404)  
```

Comment:
we could save ourselves a variable assignment by:

```suggestion
    if not object:
        return "invalid", ({"message": f"{object_type.capitalize()} {id} was not found"}, 404)  
```